docs/controls/Loading.md

```diff
@@ -0,0 +1,60 @@
+# Loading XAML Control 
+
+The loading control is for showing an animation with some content when the user should wait in some tasks of the app.
+
+## Syntax
+
+An example of how we can build the loading control.
+
+``` xaml
+<controls:Loading x:Name="LoadingControl" HorizontalAlignment="{Binding HorizontalAlignment.Value}" 
+                    VerticalAlignment="{Binding VerticalAlignment.Value}"
+                    Background="{Binding Background.Value}" Opacity="{Binding Opacity.Value}">
+    <ContentControl x:Name="LoadingContentControl"/>
+</controls:Loading>
+```
+- **HorizontalAlignment** and **VerticalAlignment** are responsible to align the custom content of the control. It aligns what we create inside as a ContentPresenter.
+- **Background** and **Opacity** are for the panel who appears and disappears behind our custom control.
+- **ContentControl** could be everything done with xaml. For example:
```

Comment:
Good to still show an example of putting content in the control, but no need to point out ContentControl. Perhaps "Use the LoadingControl to show specialized content"